.circleci/config.yml

```diff
@@ -211,6 +211,8 @@ jobs:
         command: |
           drush cc all
           drush cgen
+          drush dis -y ding_adgangsplatformen
+          drush en -y ding_user_form
```

Comment:
Yes, DDB want's it to be the default (so update hooks will enabled it). But it would be very annoying on development installations (has it requires configuration at DBC) therefor optional in the install profile.

Reply:
<img width="452" alt="ding2 #1547 - CircleCI 2019-06-11 09-34-14" src="https://user-images.githubusercontent.com/73966/59252313-21800b00-8c2c-11e9-927b-170d080f0eb7.png">

I took another look into the situation. `ding_adgangsplatformen` will be enabled by default during installation [because it is set as an default value](https://github.com/aakb/ding2/blob/43cea016fdc99babf3e33efb65a93546f9407e7a/ding2.profile#L386) but it can be disabled.

I suggest that we do not enable it during installation instead of the current situation where it is first enabled and then afterwards disabled. This makes the installation process a bit simpler and faster. The database dumps and Docker images are more usable in development situations because they do not require that a module is disabled.

This should be doable through [the arguments passed to the installer](https://github.com/ding2/ding2/blob/c17cb5a9b72394203f9e337a8f5673c5eed33337/.circleci/config.yml#L109) using something like:

```bash
drush site-install ding2 \
# stuff
ding2_module_selection_form.modules_selection='array("ding_contact", "ding_debt", "ding_dibs")' \
# more stuff

```

Reply:
Enabled ding_user_form as default for new installations and kept update hooks as requested by DDB for existing sites.